core/embed/sec/tropic/inc/sec/tropic_common.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of the Trezor project, https://trezor.io/
+ *
+ * Copyright (c) SatoshiLabs
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef TREZORHAL_TROPIC_COMMON_H
+#define TREZORHAL_TROPIC_COMMON_H
+
+typedef enum _tropic_result {
+  TROPIC_SUCCESS = 0,  // Operation completed successfully.
+  TROPIC_ERR_SESSION_START,
+} tropic_result;
```

Comment:
It seems to me that if we eliminate `tropic_handshake`, we won't need this structure anymore. However, this doesn't rule out the possibility that we might need it in the future.